### PR TITLE
REL-2450 Release SonarQube 9.7.1

### PR DESCRIPTION
**Pull request by matteo-mara-sonarsource**

### CODE REVIEW

File: charts/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -26,33 +26,11 @@ annotations:
       url: https://github.com/SonarSource/helm-chart-sonarqube/tree/master/charts/sonarqube
   artifacthub.io/changes: |
     - kind: changed
-      description: "Updated SonarQube to 9.7.0"
```
Comment (leo-geoffroy-sonarsource) [inline charts/sonarqube/Chart.yaml:29]: you can revert this change as we are not releasing from this branch

File: charts/sonarqube-dce/Chart.yaml [hunk 2]
```diff
@@ -21,27 +21,7 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "Updated SonarQube to 9.7.0"
```
Comment (leo-geoffroy-sonarsource) [inline charts/sonarqube-dce/Chart.yaml:24]: you can revert this change as we are not releasing from this branch

Comment (matteo-mara-sonarsource) [reply charts/sonarqube-dce/Chart.yaml:24]: Right, but this is correct on the branch needed for the release right?

Comment (leo-geoffroy-sonarsource) [reply charts/sonarqube-dce/Chart.yaml:24]: yes

Comment (matteo-mara-sonarsource) [reply charts/sonarqube-dce/Chart.yaml:24]: All reverted!

File: charts/sonarqube/Chart.yaml (hunk 1 above)
Comment (matteo-mara-sonarsource) [reply charts/sonarqube/Chart.yaml:29]: All reverted!

**Review by leo-geoffroy-sonarsource — APPROVED** [review]
LGTM

**Review by jCOTINEAU — APPROVED** [review]
Lgtm
